curriculum/challenges/english/25-front-end-development/lecture-working-with-links/671682dd88e461a8e2620f38.md

```diff
@@ -10,6 +10,76 @@ dashedName: what-is-the-difference-between-absolute-and-relative-paths
+
+What is the different between absolute and relative paths?
+
+A path is a string that specifies the location of a file or directory in a file system. In web development, paths let developers link to resources like images, stylesheets, scripts, and other web pages. There are absolute and relative paths - both are essential when specifying file locations within a file system. Let's look at the two so you can decide which one of them to use and when.
+
+An absolute path is a complete link to a resource. It starts from the root directory, includes every other directory, and finally the filename and extension. The "root directory" refers to the top-level directory or folder in a hierarchy.
+
+An absolute path also includes the protocol - which could be `http`, `https`, and `file` and the domain name if the resource is on the web. Here's an example of an absolute path that links to the freeCodeCamp logo:
+
+```html
+<a href="https://design-style-guide.freecodecamp.org/img/fcc_secondary_small.svg">
+  View fCC Logo
+</a>
+```
+
+In this example, the protocol is `https`, the domain name is `design-style-guide.freecodecamp.org`, and the filename is `fcc_secondary_small.svg`.
+
+Now, what if the resource you want to link to using an absolute path is on your local machine? Here's how to link to the `about.html` file with an absolute path:
+
+```html
+<p>
+  Read more on the
+  <a
+    href="/Users/user/Desktop/fCC/script-code/absolute-vs-relative-paths/pages/about.html"
+    >About Page</a
+    >
+</p>
+```
+
+It looks like this because we are going into a folder called `Users`, then into a folder called `user`, then into a folder called `Desktop`, then into a folder called `fCC`, then into a folder called `script-code`, then into a folder called `absolute-vs-relative-paths`, then into a folder called `pages` to finally get the `about.html` file.
+
+Here's what the absolute URL looks like in the browser address bar:
+
+```sh
+file:///Users/user/Desktop/fCC/script-code/absolute-vs-relative-paths/pages/about.html
+```
+
+The URL includes the protocol, `file://`. It also include the path, which looks like this: `/Users/user/Desktop/fCC/script-code/absolute-vs-relative-paths/pages/`, and represents the series of folders that lead to the file. And finally, it also includes the `about.html`, which is the filename and the extension.
+
+Now, lets look at the relative path. A relative path specifies the location of a file relative to the directory of the current file. It does not include the protocol or the domain name, making it shorter and more flexible for internal links within the same website. For example, if you want to link the `about.html` file, from a `contact.html` file in the same folder, you could do it like this:
+
+```html
+<p>
+  Read more on the
+  <a href="about.html">About Page</a>
+</p>
+```
+
+If your `about.html` file was in a `pages` folder relative to the `contact.html` file, you could like it with the relative path system like this:
```

Comment:
```suggestion
If your `about.html` file was in a `pages` folder relative to the `contact.html` file, you could link it with the relative path system like this:
```
I noticed that this example is not in the video. It's fine to add it but I'm not sure if the folder structure is clear.

Reply:
Yea, I kind of just added this part for some reason - to try and make it a little more clear. I want to give this another look to see if I can make it align with the video more.

Reply:
alright, I adjusted it to align with the video the best I could 👍